products/compute/api.yaml

```diff
@@ -732,7 +756,6 @@ objects:
         description: |
           URL of the disk type resource describing which disk type to use to
           create the disk. Provide this when creating the disk.
-        output: true
```

Comment:
Type should be a resource reference to DiskType instead of string

Reply:
I see that TODO, but we don't support resource references in Terraform yet.  Do we need to change it before adding the `labels` field?